<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="left_side"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="right_side"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="start_train"/>
    <addaction name="spelling_train"/>
    <addaction name="createList"/>
    <addaction name="open_list"/>
    <addaction name="add_word"/>
    <addaction name="change_list"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="settings"/>
    <addaction name="network_settings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="createList">
   <property name="text">
    <string>Добавить коллекцию</string>
   </property>
  </action>
  <action name="open_list">
   <property name="text">
    <string>Открыть коллекцию</string>
   </property>
  </action>
  <action name="add_word">
   <property name="text">
    <string>Добавить слово</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Начать тренировку</string>
   </property>
  </action>
  <action name="start_train">
   <property name="text">
    <string>Начать тренировку</string>
   </property>
  </action>
  <action name="change_list">
   <property name="text">
    <string>Изменить коллекцию</string>
   </property>
  </action>
  <action name="spelling_train">
   <property name="text">
    <string>Тренировка написания</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>Упражнения</string>
   </property>
  </action>
  <action name="network_settings">
   <property name="text">
    <string>Сеть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
